src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -54,7 +55,6 @@
             c.custom_command('restore', 'db_restore', no_wait_param='raw')
             c.command('show', 'get')
             c.custom_command('list', 'db_list')
-            c.command('list-usages', 'list_usages')
```

Comment:
why remove this?

Reply:
I think github is being dumb about merges. It should look right when I rebase.

Reply:
Never mind, it is really deleted, however sql db list-usages is redefined further down. That later one is the correct one.